diff --git a/index.d.ts b/index.d.ts
index 9cd22baa266c35d5109117ef2ae3e737b2789e06..590a779eca8009472aabc86f8e1a6d941db2b0da 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -7,7 +7,7 @@ export enum FLOAT32_OPTIONS {
 
 export interface Options {
 	useFloat32?: FLOAT32_OPTIONS
-	useRecords?: boolean | (value:any)=> boolean
+	useRecords?: boolean | ((value:any)=> boolean)
 	structures?: {}[]
 	moreTypes?: boolean
 	sequential?: boolean
